site/content/rfds/0007-simplified-release-procedures.md

```diff
@@ -0,0 +1,164 @@
+---
+title: "Simplified Release Procedures"
+---
+
+## The Current Situation
+
+Currently, the Hipcheck _project_ has the following artifacts we release
+periodically.
+
+- `hc`: The main Hipcheck binary, produced by the `hipcheck` package.
+- `hc-update`: The Hipcheck self-updater, generated by `cargo-dist`.
+- `hipcheck-macros`: A Hipcheck-internal library.
+- Docker image: published to Docker Hub.
+
+These are published to the following places:
+
+- `hc`:
+  - Crates.io (as `hipcheck`)
+  - GitHub Releases w/ install script generated by `cargo-dist`
+- `hc-update`:
+  - GitHub Releases w/ install script generated by `cargo-dist`
+`- `hipcheck-macros`:
+  - Crates.io
+- Docker image:
+  - Docker Hub
+
+When we cut a new release of `hc`, the flow has been:
+
+1. Update the `CHANGELOG.md` for the project with release notes for the new
+   version, open a PR, and get that merged.
+2. Run `cargo release` locally to make a new release of `hc` to Crates.io.
+   At the end of that process, `cargo release` creates a publishes a tag for
```

Comment:
`creates a publishes a` --> `creates and publishes a`

Reply:
Fixed!